remoting/src/main/java/org/apache/rocketmq/remoting/protocol/RemotingCommand.java

```diff
@@ -61,10 +61,13 @@
     static {
         final String protocol = System.getProperty(SERIALIZE_TYPE_PROPERTY, System.getenv(SERIALIZE_TYPE_ENV));
         if (!isBlank(protocol)) {
+            byte protocolByte = -1;
             try {
-                serializeTypeConfigInThisServer = SerializeType.valueOf(protocol);
+                protocolByte = (byte) Integer.parseInt(protocol);
+                System.out.println(protocolByte);
+                serializeTypeConfigInThisServer = SerializeType.valueOf(protocolByte);
             } catch (IllegalArgumentException e) {
-                throw new RuntimeException("parser specified protocol error. protocol=" + protocol, e);
+                throw new RuntimeException("parser specified protocol error. protocol=" + serializeTypeConfigInThisServer, e);
             }
```

Comment:
IMO, No need to catch the runtime exception and rethrow it

Reply:
Okay,I will fix it.